_extras/Recipe.md

```diff
@@ -0,0 +1,75 @@
+---
+layout: page
+title: "Online Workshop Recipe"
+---
+
+# A Recipe for an Online Carpentries Workshop
+
+In the past year+ of teaching online, our community has learned many things. Our initial [online training recommendations](https://carpentries.org/online-workshop-recommendations/#planning-your-workshop) recognised that we were in uncharted territory for Carpentries workshops! In consulting with community members, we heard many different narratives and suggestions. Rather than creating a single 'recipe,' we chose to distill that advice into 'things to think about', giving Instructors space to engineer their own solutions to the challenges presented by teaching online.
```

Comment:
I think we should specifically say up front that these are recommendations for teaching technical workshops (or workshops on data and computing skills), since some related ideas (e.g., workflow dependencies) are described later, and because we don't tend to have some features (like helpers) in instructor training.